pkg/k8sintrospect/collector.go

```diff
@@ -408,6 +416,7 @@ func (c *collector) Run() error {
 		for _, objCfg := range c.cfg.Objects {
 			objClient := clientMap[objCfg.GVK]
 			if objClient == nil {
+				log.Warningf("Error generating diagnostic bundle. No client found for %s", objCfg.GVK.String())
```

Comment:
Hmm this is optional for me but: it would be great to include this information within the `errors.txt` file by adding it to the errors list within the collector. This way we have this information more clearly depicted within the diagnostic bundle.